src/common/boundary_condition.cpp

```diff
@@ -46,26 +47,27 @@ void BoundaryCondition::project(mfem::ParGridFunction& gf, const mfem::ParFinite
     // Generate the scalar dof list from the vector dof list
     SLIC_ASSERT_MSG(space, "Only BCs associated with a space can be projected.");
     mfem::Array<int> dof_list(size);
-    std::transform(&tdofs[0], &tdofs[0] + size, &dof_list[0], [&space](int tdof) { return space.VDofToDof(tdof); });
+    std::transform(&tdofs[0], &tdofs[0] + size, &dof_list[0],
```

Comment:
Can you add a unit test to ensure that mfem::Array continues to work with std::transform?  This could be a very bad time if this changes from under us.

Reply:
Especially since we use different C++ standards than they do.